eng/common/cross/toolchain.cmake

```diff
@@ -193,7 +193,7 @@ if(TARGET_ARCH_NAME MATCHES "^(arm|armel)$")
 
   add_definitions (-DCLR_ARM_FPU_CAPABILITY=${CLR_ARM_FPU_CAPABILITY})
 
-  if(TARGET_ARCH_NAME STREQUAL "armel")
+  if(DEFINED TIZEN_TOOLCHAIN AND TARGET_ARCH_NAME STREQUAL "armel")
```

Comment:
this change in eng/common will need to be done in https://github.com.dotnet/arcade since these files are mirrored from there and changes would be reverted in the next arcade update.